secretmanager/src/test/java/com/example/SnippetsIT.java

```diff
@@ -47,6 +47,7 @@
 /** Integration (system) tests for {@link Snippets}. */
 @RunWith(JUnit4.class)
 public class SnippetsIT {
+  private static final String IAM_USER = "user:sethvargo@google.com";
```

Comment:
probably should change this

Reply:
Hi @averikitsch - is there a good service account that exists (or could you create one) in the test project please?

Reply:
iam-samples@java-docs-samples-testing.iam.gserviceaccount.com. Currently it has no roles or keys associated with it.

Reply:
Done